pkg/polaris/creater.go

```diff
@@ -196,6 +199,22 @@ func GetPolarisBaseSecrets(polaris Polaris) (map[string]runtime.Object, error) {
 	return mapOfUniqueIDToBaseRuntimeObject, nil
 }
 
+func GetVersions(baseURL string) ([]string, error) {
+	serverURL, err := url.Parse(baseURL)
+	if err != nil {
+		return nil, err
```

Comment:
Can we add a little more verbosity to why this error may have occured?

Reply:
The error message has been updated.